src/main/java/iudx/catalogue/server/apiserver/GeocodingApis.java

```diff
@@ -24,8 +25,10 @@ public final class GeocodingApis {
 
   private GeocodingService geoService;
   private static final Logger LOGGER = LogManager.getLogger(GeocodingApis.class);
-
-  public GeocodingApis() {}
+  private Api api;
```

Comment:
Similarly in these subsequent classes

Reply:
Since api instance is being used in RatingApis, RelationshipApis, SearchApis, I thought this api instance could be used in the future in GeocodingApis, CrudApis.